src/pages/Spacing.vue

```diff
@@ -0,0 +1,304 @@
+<script setup>
+// Note there's a way we can just import these globally so we don't have to do it in every page
+import Box from '../components/Box.vue'
+import Container from '../components/Container.vue'
+import WidthController from '../components/WidthController.vue'
```

Comment:
Personally I would expose these as named exports from a `index.js` living under the components folder and setup a components alias so that you can import them as 👇 

```js
import { Foo, Bar, Baz } from 'components'
```

feel free to disagree, but it would make things a little more pretty (this can be added later if its not within the current scope of things)

Reply:
These are going to be used often enough they should probably just be 'magic' and globally available TBH - but yeah improving ergonomics here is a great thought